src/main/scala/training/modules/shops/ShopRepository.scala

```diff
@@ -0,0 +1,15 @@
+package training.modules.shops
+
+class ShopRepository {
+
+  def createShop(businessName: String, name: String, activityId: Int, stratumId: Int, address: String,
```

Comment:
Why create this repository in the main module? 
This behavior could be useful for testing (but better take a look to ScalaCheck).
If you decide to leave it this way, I suggest to make changes so you can have the GraphQL interface independent from the implementation.